provision/bootstrap.sh

```diff
@@ -51,14 +49,14 @@ apt-get -y install php$PHP_VERSION php$PHP_VERSION-dev php-redis php-apcu php$PH
 apt-get -y install ffmpeg imagemagick pngquant gifsicle freeglut3 webp sphinxsearch
 
 
-# TODO:: is this needed? were already using php 7
-# Switch Apache to PHP 7
+# Switch Apache to correct php version
+a2dismod php7.0 #disable existing php module
 a2enmod php$PHP_VERSION
-
+service apache2 restart
 
 echo "\nConfiguring PHP"
 # Copy config change setting and copy back
-sed "s/opcache\.enable.*/opcache.enable = 1/" /etc/php/$PHP_VERSION/apache2/conf.d/user.ini > /tmp/php.edited.user.ini
+sed "s/opcache\.enable.*/opcache.enable = 1/" /etc/php/7.0/apache2/conf.d/user.ini > /tmp/php.edited.user.ini
```

Comment:
This might be why memcache and redis aren't found

Reply:
Not exactly. Once PHP 7.0 is uninstalled those should no longer exist. 

If anything, with newer Ubuntu's and PHP opcache shouldn't need to manually enabled anymore. All it requires is the `php7.1-opcache` package and it has everything it needs.